include/dcmqi/itkTestMain.h

```diff
@@ -89,7 +89,7 @@ int main(int ac, char *av[])
 {
   itk::FloatingPointExceptions::Enable();
 
-  double       intensityTolerance  = 2.0;
+  double       intensityTolerance  = 0.0001;
```

Comment:
Especially for comparing segmentations - they can be very different pixel-wise, but test would still succeed!